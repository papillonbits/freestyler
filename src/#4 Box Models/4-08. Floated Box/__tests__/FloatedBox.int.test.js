import { regular } from './FloatedBox.int.story'

describe('<FloatedBox />', () => {
  describe('Render', () => {
    test('must match regular()', () => {
      expect(global.renderToJSON(regular())).toMatchSnapshot()
    })
  })
})
