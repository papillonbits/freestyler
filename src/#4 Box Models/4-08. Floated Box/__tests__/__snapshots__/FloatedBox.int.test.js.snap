// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FloatedBox /> Render must match regular() 1`] = `
<div>
  <div
    class="floated-box"
  >
    <h1>
      Floated Box
    </h1>
    <div
      class="container"
    >
      <div
        class="default"
      >
        BEFORE
      </div>
      <div
        class="box small"
      >
        FLOAT BEFORE
      </div>
      <div
        class="box"
      >
        ↑ 
        <br />
         Top 
        <br />
         ← Left       Right → 
        <br />
         Bottom 
        <br />
         ↓
         
      </div>
      <div
        class="box small"
      >
        FLOAT AFTER
      </div>
      <div
        class="default"
      >
        AFTER
      </div>
    </div>
  </div>
</div>
`;
